src-django/api/models.py

```diff
@@ -34,6 +34,34 @@ class Meta:
         ordering = ['procedure', 'display_index']
 
 
+class Concept(models.Model):
+    TYPES = (
+        ('string', 'string'),
+        ('boolean', 'boolean'),
+        ('number', 'number'),
+        ('complex', 'complex')
+    )
+
+    uuid = models.UUIDField(default=uuid.uuid4, null=False, blank=False, editable=False)
```

Comment:
can we enforce proper constraints here? Everything that is blankable/nullable should be and everything that isn't, shouldn't.